packages/core/src/DataSourceContent/useStyles.js

```diff
@@ -1,6 +1,6 @@
-import { makeStyles } from "@material-ui/core/styles";
+import { makeStyles } from 'tss-react/mui';
```

Comment:
`tss-react`?

Reply:
This was the best option I found that would not require a lot of changes. It is also part of the [docs](https://mui.com/material-ui/migration/migration-v4/#:~:text=%40material%2Dui/core%20%2D%3E%20%40mui/material)